datacommons_client/utils/response.py

```diff
@@ -0,0 +1,311 @@
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List
+
+from datacommons_client.models.node import Arcs, NextToken, NodeDCID, Properties
+from datacommons_client.models.observation import (
+    Facet,
+    Variable,
+    facetID,
+    variableDCID,
+)
+from datacommons_client.models.resolve import Entity
+from datacommons_client.models.sparql import Row
+
+
+@dataclass
+class DCResponse:
+    """Represents a structured response from the Data Commons API."""
+
+    json: Dict[str, Any] = field(default_factory=dict)
+
+    def __repr__(self) -> str:
+        """Returns a string with the name of the object,"""
+        return f"<Raw Data Commons API response>"
+
+    @property
+    def next_token(self):
+        return self.json.get("nextToken")
+
+
+@dataclass
+class NodeResponse:
+    """Represents the response from the Data Commons API.
```

Comment:
Can you note the specific endpoint that returns this response? (and for other *Response objects in this file)

Reply:
Improved docstrings: [b166513](https://github.com/datacommonsorg/api-python/pull/204/commits/b166513fa89d12012426457bfafeee649f31d930)